src/utils/normalizer.js

```diff
@@ -1,4 +1,9 @@
-import { __DEV__ } from 'react-native';
```

Comment:
`__DEV__` is used in that file, but import is removed. I guess it may be a part of RN globals, but wanted to recheck if it's intentionally.

Reply:
Yes, this was intentional because the ESLinter complains about the import